packages/calypso-stripe/package.json

```diff
@@ -0,0 +1,52 @@
+{
+	"name": "@automattic/calypso-stripe",
+	"version": "1.0.0",
+	"description": "A set of helper functions and components for using Stripe on WordPress.com",
+	"main": "dist/cjs/index.js",
+	"module": "dist/esm/index.js",
+	"types": "dist/types/index.d.ts",
+	"sideEffects": false,
+	"scripts": {
+		"clean": "npx rimraf dist \"../../.tsc-cache/packages__calypso-stripe*\"",
+		"prepare": "tsc --project ./tsconfig.json && tsc --project ./tsconfig-cjs.json",
+		"prepublish": "yarn run clean",
+		"watch": "tsc --project ./tsconfig.json --watch"
```

Comment:
As you mention, it would be nice to have the improvements from #47002 from the start. Create two tasks, `build:esm` and `build:cjs`. In `prepare`, run only `build:esm`. In `prepack`, clean and run both. No `prepublish` script.

Reply:
Got it. I see a lot of `copy-assets` in other packages. What does that do, and do we need it here?

Reply:
`copy-assets` copies `.scss` files or images from `src/` to the `dist/` directory. The transpile scripts only "copy" (i.e., transpile) the JS and TS files. If we need anything else in the published package, we need `copy-assets`.

Reply:
Perfect. Thanks. This package has no assets, so we should be good.